static/src/javascripts/bootstraps/video-embed.js

```diff
@@ -16,7 +15,7 @@ define([
     'common/modules/video/fullscreener',
     'common/views/svgs',
     'text!common/views/ui/loading.html',
```

Comment:
Could we also update this one to use the loader? I wonder if you've missed any others, might be worth doing a project wide search for `text!foo.html`.